app/views/templates/header.jst

```diff
@@ -0,0 +1,16 @@
+<h3><%= title %> <em>(<%= count %>)</em></h3>
+<% if (title == "Up Next") { %>
+    <div class="button" id="reset-upnext">
```

Comment:
You should avoid to use too much `div`s and rely upon the markup semantics: if you need a button, then juste use a `<button>` tag.